examples/io-csv-datamapping/io_csv.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina/io;
+
+
+// Define the record to bind the data with
+type Employee record{
+    int id;
+    string name;
+    int salary;
+};
+
+public function main() returns error? {
+    // Initializes the CSV file path and content.
+    string csvFilePath1 = "./files/csvFile1.csv";
+    string csvFilePath2 = "./files/csvFile2.csv";
+    Employee[] csvContent = [{id:1, name:"James", salary:10000}, {id:2, name:"Nathan", salary:150000},
+    {id:3, name:"Ronald", salary:120000}, {id:4, name:"Roy", salary:6000},
+    {id:5, name:"Oliver", salary:1100000}];
+
+    // Writes the given content record[] to a CSV file.
+    check io:fileWriteCsv(csvFilePath1, csvContent);
+    // Reads the previously-saved CSV file as a record[].
+    Employee[] readCsv = check io:fileReadCsv(csvFilePath1);
+    io:println(readCsv);
+
+    // Writes the given content as a stream to a CSV file.
+    check io:fileWriteCsvFromStream(csvFilePath2, readCsv.toStream());
+    // Reads the previously-saved CSV file as a record stream.
```

Comment:
```suggestion
    // Reads the previously-saved CSV file as a record stream.
```
The lines before have `record[]` instead of saying `record stream`. Let's be consistent if both are referring to the same.

Reply:
No. There are two APIs to support `record[]` and `record streams` in each scenario(reading and writing). In this BBE we have demonstrated both so these two are not referring to the same